docs: Improve Cognito documentation for social providers with Cognito (#35909)
expo/expo: docs/pages/guides/authentication.mdx

}
```

> - To open a specific Social Provider such as Google you can supply the providers name with `identity_provider` parameter within extraParams for the signIn url.

Reviewer: ```suggestion
- To open a specific social identity provider, such as Google, you can supply the provider's name with the `identity_provider` parameter within `extraParams` for the sign in URL.
```

```tsx
extraParams: {
    identity_provider: "Google",
}
```

> - Valid providers are `Google`, `COGNITO`, `Facebook` `LoginWithAmazon`, `SignInWithApple`. These are case sensitive and must match Social Provider within Cognito. You can provide custom providers.

Reviewer: ```suggestion
- Valid providers are `Google`, `COGNITO`, `Facebook` `LoginWithAmazon`, `SignInWithApple`. These are case sensitive and must match the social provider within Cognito. You can provide custom providers.
```

> - Valid providers are `Google`, `COGNITO`, `Facebook` `LoginWithAmazon`, `SignInWithApple`. These are case sensitive and must match the social provider within Cognito. You can provide custom providers.

Reviewer: ```suggestion

- Valid providers are `Google`, `COGNITO`, `Facebook` `LoginWithAmazon`, `SignInWithApple`. These are case sensitive and must match the social provider within Cognito. You can provide custom providers.
```
